packages/shared/src/modules/categories/libs/enums/categories-api-path.enum.ts

```diff
@@ -1,4 +1,5 @@
 const CategoriesApiPath = {
+	GET_BY_ID: "/get-by-id",
```

Comment:
I'm not sure about that api path name. I guess it is better not to have "get" or any other http method mentioned in api pathes.